share/completions/iconutil.fish

```diff
@@ -0,0 +1,3 @@
+complete -c iconutil -s c -l "convert" -r -f --arguments "icns iconset" -d "Convert to the specified type."
+complete -c iconutil -s o -l "output" -r -F --description "Overrides the default output file name."
+complete -c iconutil -r -F -d "Source file"
```

Comment:
> the optional second positional argument should not use file completions.

using something like argparse (see examples in share/completions) would be a way to avoid such false positives but I guess it's not worth it here